static/css/style.css

```diff
@@ -1116,6 +1116,12 @@ div#container-display-show {
     width: 300px;
 }
 
+#show-selector {
+    width: auto;
+    height:25px;
+    padding:1px;
+}
+
```

Comment:
Probably getting overridden by bootstrap.
Try using `!important`, even though I read that it's unrecommended.